main.nf

```diff
@@ -26,6 +25,22 @@ params.fasta = WorkflowMain.getGenomeAttribute(params, 'fasta')
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
+include { validateParameters; paramsHelp } from 'plugin/nf-validation'
+
+// Print help message if needed
+if (params.help) {
+    def logo = NfcoreTemplate.logo(workflow, params.monochrome_logs)
+    def citation = '\n' + WorkflowMain.citation(workflow) + '\n'
+    def String command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --genome GRCh37 -profile docker"
```

Comment:
Maybe also a question for @jfy133 but why `genome GRCh37`

Reply:
Ah that's template! we should update that command to our very basic example:

```
nextflow run <...> --input XYZ --databases CYZ -profile docker --run_kraken2
```

for example

Reply:
But shall we leave it as it is for now?

Reply:
No should be updated here!

Reply:
How about the one from the usage.md: 

`nextflow run nf-core/taxprofiler --input samplesheet.csv --databases databases.csv --outdir <OUTDIR> -profile docker --run_<TOOL1> --run_<TOOL2>`

Reply:
Yes!

Reply:
Done in #366